apps/web/pages/api/auth/verify-email.ts

```diff
@@ -101,6 +103,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       },
     });
 
+    const stripeCustomerId = await getStripeCustomerIdFromUserId(user.id);
+    await stripe.customers.update(stripeCustomerId, { email: updatedEmail });
```

Comment:
We need to keep self-hosters in mind. We should only make this call if Stripe is enabled. We should also make the stripe customer call only if a stripe customer id is found.

Reply:
Good catch @joeauyeung, could you please re-review when you have a minute?